.github/workflows/docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docs/**'
+      - 'mkdocs.yml'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  build-and-deploy:
+    if: ${{ github.repository == 'opencadc/deployments' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+        with:
+          fetch-depth: 1
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install MkDocs
```

Comment:
Use `uv` to create a python package called deployments, and within that package add dependencies for docs. e.g.

```bash
uv init --lib deployments
uv add --group=docs mkdocs-material something-other-dep
```

Then the build command becomes,
```
uv sync --group docs
uv run mkdocs serve
uv run mkdocs build
```

We want to do this because sync dependencies across OS's and architectures.